### DISCUSSION TITLE
SignServer for Document Signing with EJBCA

### DISCUSSION THREAD

**Discussion by hzami** [Keyfactor/signserver-ce#84] [Q&A] [answered]
Hello community,
I am working on document signing use case using SignServer and EJBCA (ce) . Can you please help me with the steps to use/deploy SignServer for signing documents? I couldn't find any tutorial or guide that explains how to do this.
Thank you!

**Comment by huongdang0208** [comment, marked as answer]
Hi, I am just following the tutorials of KeyFacter Community and they guide me through 2 steps:

1.  You need to issue a client certificate to access to signserver, and you can get these by EJBCA, here the guide video: https://www.youtube.com/watch?v=wMD1GgSF-JE
2. When you finish step 1 you have got ManagementCA.pem and your p12 file. You use your ManagementCA.pem to start signserver like this doc: https://hub.docker.com/r/keyfactor/signserver-ce. Open your localhost you just started up, then import your p12 file you've got into the browser to access the signservre
Hope you doing successfully!
